x-pack/plugin/src/yamlRestTest/resources/rest-api-spec/test/ml/trained_model_cat_apis.yml

```diff
@@ -78,17 +79,17 @@ setup:
         model_id: a-regression-model-0
   - match:
       $body: |
-        / #id                           heap_size    operations    create_time    ingest.pipelines  data_frame.id
-        ^ (a\-regression\-model\-0 \s+  \w+      \s+ \d+       \s+ .*?        \s+ \d+               .*?          \n)+  $/
+        / #id                           heap_size    operations    create_time  type  ingest.pipelines  data_frame.id
+        ^ (a\-regression\-model\-0 \s+  \w+      \s+ \d+       \s+ .*?        \s+ \d+               .*?    \n)+  $/
 
   - do:
       cat.ml_trained_models:
         v: true
         model_id: a-regression-model-0
   - match:
       $body: |
-        /^  id                       \s+  heap_size \s+ operations \s+ create_time \s+ ingest\.pipelines \s+ data_frame\.id \n
-           (a\-regression\-model\-0  \s+  \w+       \s+ \d+        \s+ .*?         \s+ \d+               \s+ .*?            \n)+        $/
+        /id\s+heap_size\s+operations\s+create_time\s+type\s+ingest\.pipelines\s+data_frame\.id\s*\n
+        (a\-regression\-model\-0\s+\w+\s+\d+\s+\S+\s+tree_ensemble\s+\d+\s+\w+\n)+$/
```

Comment:
spaces are there for readability.

Reply:
It may be more readable but it makes the test very brittle. I've found this is still quite readable as each column is delineated by a `\s+` and the regex is easer to grok.

Reply:
> It may be more readable but it makes the test very brittle. 

I don't understand how it makes the test more brittle. Tests don't fail more often or not due to the spaces and to me, it seems WAY easier to know what column my current regex clause is messing with when there are white spaces.